### ISSUE TITLE
[suggestion] smart chute as reward for the sifter quest

### ISSUE THREAD

**Issue by Quezler** [Laskyyy/Create-Astral#186]
Trying to pull items out of the sifter via hopper/chute is a bit troublesome since the block has no separate input/output inventory and will simply instantly output e.g. the gravel.

To remedy this one could use a hopper with filtered slots, but that doesn't scale well with several sievables nor can you look at the block to see the items inside of it.

I suggest giving the player a single smart chute as a reward for this quest, so they can filter it on blacklist mode.

Currently the quest has no rewards whatsoever, and i think this would be a nice QOL:

![Screenshot 2023-07-06 at 12 13 34](https://github.com/Laskyyy/Create-Astral/assets/3179271/83742978-c452-4832-b141-08583ae52d7e)

**Comment by Quezler** [comment]
Yes i did come across https://github.com/oierbravo/createsifter/issues/30 suggesting the storage units, but just like the hopper you'll run into slot/side limitations.

Happy to hear a solution is being cooked up in house, consider this suggestion merely a bandaid solution then. 👍

**Comment by The-Shortman** [comment]
Sifter has indeed been removed in 2.1
